plugins/edit-status.yaml

```diff
@@ -20,20 +20,39 @@ spec:
         matchLabels:
           os: darwin
           arch: amd64
-      uri: https://github.com/ulucinar/kubectl-edit-status/releases/download/v0.1.0/kubectl-edit-status_v0.1.0_darwin_amd64.tar.gz
-      sha256: 5dfef266c8db616be3296ebaab5e7cdef85e447c89c7a4588e9f2e065b11c856
+      uri: https://github.com/ulucinar/kubectl-edit-status/releases/download/v0.2.0/kubectl-edit-status_v0.2.0_darwin_amd64.tar.gz
+      sha256: dbb65ab56beb3ece575748c17961e76018ff2e3eb74a4d221369b308d049f8e3
+      files:
+        - from: "*"
+          to: "."
```

Comment:
This is the default. Could you strip those File specs from the PR, then this will be auto-approved.

Reply:
Hi @corneliusweig,
Thank you very much for the hint. I have removed the `files` sections from all `platform`s but the PR is not still auto-approved. Could it be because I also add a new platform entry for darwin_arm64?

Reply:
Yes, that's expected :)
But when such manual intervention is necessary, it's always a good opportunity to clean up.

Reply:
Thank you @corneliusweig, agreed. v0.2.0 is now installable on Apple Silicon machines.